test/upgrade/upgrade_test.go

```diff
@@ -129,7 +133,9 @@ func postUpgradeTests(ctx *test.Context) []pkgupgrade.Operation {
 	tests = append(tests, eventingupgrade.PostUpgradeTests()...)
 	tests = append(tests, kafkaupgrade.ChannelPostUpgradeTest(),
 		kafkaupgrade.SourcePostUpgradeTest(),
-		kafkabrokerupgrade.BrokerPostUpgradeTest())
+		kafkabrokerupgrade.BrokerPostUpgradeTest(),
+		kafkabrokerupgrade.SinkPostDowngradeTest(),
```

Comment:
Why "PostDowngrade" and not "PostUpgrade" ? I can see they're same but then it's confusing in logs. We're only doing upgrades, not downgrades in Serverless operator.

Reply:
Yes, `WIP`